zuul.d/jobs.yaml

```diff
@@ -108,3 +108,12 @@
       nodes:
         - name: builder
           label: ubuntu-xenial-builder
+
+- job:
+    name: contrail-vnc-docker-provision-centos74
+    voting: False
+    run: playbooks/docker/centos74-provision
```

Comment:
This playbook should probably be `playbooks/contrail/centos74-provision-systest` or similar - this job will both provision and run tests.